src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/Design/DesignerVerb.cs

```diff
@@ -25,7 +31,12 @@ public DesignerVerb(string text, EventHandler handler) : base(handler, StandardC
         /// </summary>
         public DesignerVerb(string text, EventHandler handler, CommandID startCommandID) : base(handler, startCommandID)
         {
-            Properties["Text"] = text == null ? null : Regex.Replace(text, @"\(\&.\)", "");
+            Properties["Text"] = ReplaceIfNotNull(text);
+        }
+
+        private static string? ReplaceIfNotNull(string? text)
+        {
+            return text == null ? null : DesignerVerbRegex.GetParameterReplacementRegex().Replace(text, "");
         }
```

Comment:
We shouldn't need to introduce a new helper for this.  Instead, `public DesignerVerb(string text, EventHandler handler) : base(handler, StandardCommands.VerbFirst)` can be changed to `public DesignerVerb(string text, EventHandler handler) : this(text, handler, StandardCommands.VerbFirst)`, and then all the logic remains in the other ctor.